src/CodeXMLParse.cpp

```diff
@@ -46,8 +45,67 @@ void AnnotateOpref(ANNOTATOR_PARAMS)
 	annotation.type = R_CODE_ANNOTATION_TYPE_OFFSET;
 	annotation.offset.offset = op->getAddr().getOffset();
 }
+void AnnotateFunctionName(ANNOTATOR_PARAMS)
+{
+	const char *func_name = node.child_value();
+	if(!func_name)
+		return;
+	RCodeAnnotation annotation = {};
+	annotation.type = R_CODE_ANNOTATION_TYPE_FUNCTION_NAME;
+	pugi::xml_attribute attr = node.attribute("opref");
+	if(attr.empty())
+	{
+		if(ctx->func->getName() == func_name)
+		{
+			annotation.function_name.name = strdup(ctx->func->getName().c_str());
+			annotation.function_name.offset = ctx->func->getAddress().getOffset();
+			// Code below makes an offset annotation for the function name(for the currently decompiled function)
+			RCodeAnnotation offsetAnnotation = {};
+			offsetAnnotation.type = R_CODE_ANNOTATION_TYPE_OFFSET;
+			offsetAnnotation.offset.offset = annotation.function_name.offset;
+			out->push_back(offsetAnnotation);
```

Comment:
With the currently pushed commits, I have also implemented offset annotation for the currently decompiled function's name. See gif
![offset_annotation_for_function_name](https://user-images.githubusercontent.com/18501167/86903867-8b1f6100-c12d-11ea-9db6-8a77e44c1efc.gif)